src/components/DeeplinkWrapper/index.website.js

```diff
@@ -0,0 +1,155 @@
+            focused = false;
+        });
+
+        setTimeout(() => {
+            if (!focused) {
+                this.setState({appInstallationCheckStatus: 'installed'});
+            } else {
+                this.setState({appInstallationCheckStatus: 'not-installed'});
+            }
+        }, 500);
+
+        // check if pathname matches with deeplink routes
+        const pathname = window.location.pathname;
+        const matchedRoute = _.find(deeplinkRoutes, (route) => {
+            const routeRegex = new RegExp(route.pattern);
+            return routeRegex.test(pathname);
+        });
+
+        if (matchedRoute) {
+            this.setState({deeplinkMatch: true});
+            this.openRouteInDesktopApp();
+        } else {
+            this.setState({deeplinkMatch: false});
+        }
+    }
+
+    openRouteInDesktopApp() {
+        const pathname = window.location.pathname;
+        const expensifyUrl = new URL(CONFIG.EXPENSIFY.NEW_EXPENSIFY_URL);
+        const expensifyDeeplinkUrl = `${CONST.DEEPLINK_BASE_URL}${expensifyUrl.host}${pathname}`;
+
+        // This check is necessary for Safari, otherwise, if the user
+        // does NOT have the Expensify desktop app installed, it's gonna
+        // show an error in the page saying that the address is invalid
+        if (CONST.BROWSER.SAFARI === Browser.getBrowser()) {
+            const iframe = document.createElement('iframe');
+            iframe.style.display = 'none';
+            document.body.appendChild(iframe);
+            iframe.contentWindow.location.href = expensifyDeeplinkUrl;
+
+            setTimeout(() => {
+                if (!iframe.parentNode) {
+                    return;
+                }
+
+                iframe.parentNode.removeChild(iframe);
+            }, 100);
+        } else {
+            window.location.href = expensifyDeeplinkUrl;
+        }
+    }
+
+    isMacOSWeb() {
+        return !Browser.isMobile() && (
+            typeof navigator === 'object'
+            && typeof navigator.userAgent === 'string'
+            && /Mac/i.test(navigator.userAgent)
+            && !/Electron/i.test(navigator.userAgent)
+        );
+    }
+
+    render() {
+        if (this.state.appInstallationCheckStatus === 'checking') {
+            return <FullScreenLoadingIndicator style={styles.flex1} />;
+        }
+
+        if (
+            this.state.deeplinkMatch
+            && this.state.appInstallationCheckStatus === 'installed'
+        ) {
+            return (
+                <View style={styles.deeplinkWrapperContainer}>
+                    <View style={styles.deeplinkWrapperMessage}>
+                        <View style={styles.mb2}>
+                            <Icon
+                                width={200}
+                                height={164}
+                                src={Illustrations.RocketBlue}
+                            />
+                        </View>
+                        <Text style={styles.textXXLarge}>
```

Comment:
@cowboycito can you please update this to use the new `headlineFont`?

Reply:
Done

Reply:
This is how it's looking with the font change:

<img width="1884" alt="Screen Shot 2023-01-06 at 17 48 35" src="https://user-images.githubusercontent.com/47456024/211097410-29b617b9-32ad-4ca0-8220-1859027db374.png">

Reply:
Looks great!